src/components/interface/cbuf_mgr/cbuf_c.c

```diff
@@ -136,7 +136,10 @@ __cbufp_alloc_slow(int cbid, int size, int *len, int *error)
 	/* Nothing collected...allocate a new cbuf! */
 	if (amnt == 0) {
 		cbid = cbuf_create(cos_spd_id(), size, cbid*-1);
-		if (cbid == 0) assert(0);
+		if (cbid == 0) 
+        {
+            assert(0);
+        }
```

Comment:
Yeah... sorry about that. Gabe caught me expanding tabs to spaces and we decided that wasn't the way to go. Must have missed this one.